actors/migration/nv16/test/miner_migration_test.go

```diff
@@ -146,11 +150,48 @@ func createMinersAndSectorsV7(t *testing.T, ctx context.Context, ctxStore adt.St
 	return append(minersToProve, minerInfos...), v
 }
 
+func makeManifest(t *testing.T, store cbor.IpldStore) cid.Cid {
```

Comment:
`makeFakeManifest` `makeTestCIDsManifest`.. something to indicate its not real

Reply:
ok.

Reply:
I'll call it `makeTestManifest`.